### PR TITLE
Firebase auth

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/features/authentication/components/LogisterScreen.tsx [hunk 1]
```diff
@@ -31,75 +33,137 @@ const LogisterScreen = ({ }: LogisterScreenProps): JSX.Element => {
...
+                    console.log('User account signed in!')
+                })
+                .catch((error: FirebaseAuthTypes.NativeFirebaseAuthError) => {
+                    errorHandler(error)
+                })
+        }
+    }
+
+    const errorHandler = (error: FirebaseAuthTypes.NativeFirebaseAuthError) => {
+        let msg = 'oops! something went wrong'
```
Comment (chengoM123) [inline src/features/authentication/components/LogisterScreen.tsx]: you can use dictionary

File: src/features/authentication/components/LogisterScreen.tsx [hunk 2]
```diff
@@ -31,75 +33,137 @@ const LogisterScreen = ({ }: LogisterScreenProps): JSX.Element => {
...
+
+        if (error.code === 'auth/email-already-in-use')
```
Comment (chengoM123) [inline src/features/authentication/components/LogisterScreen.tsx]: add the errors to strings file

File: src/features/authentication/components/LogisterScreen.tsx [hunk 3]
```diff
@@ -31,75 +33,137 @@ const LogisterScreen = ({ }: LogisterScreenProps): JSX.Element => {
...
+                                error={emailError}
+                                setError={setEmailError}
+                            />
+                            <AppInput
+                                value={password}
+                                setValue={setPassword}
+                                placeholderText={passwordPlaceholder}
+                                contentType={ContentType.password}
+                                validate={validatePassword}
+                                styleProps={{ marginBottom: 24 }}
```
Comment (chengoM123) [inline src/features/authentication/components/LogisterScreen.tsx]: use style object

File: src/features/counter/counterSlice.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
-import {createAsyncThunk, createSlice, PayloadAction} from '@reduxjs/toolkit';
-import {RootState} from '../../app/store';
-import {fetchCount} from './counterAPI';
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { RootState } from '../../state/store'
+import { fetchCount } from './counterAPI'
 
 interface CounterState {
```
Comment (chengoM123) [inline src/features/counter/counterSlice.ts]: do you need this slice?

File: src/features/authentication/components/LogisterScreen.tsx (hunk 2 above)
Comment (leeshos) [reply src/features/authentication/components/LogisterScreen.tsx:77]: I just moved the whole errorHandler to its own file, is it still necessary to move the errors to strings file?

**Review by chengoM123 — APPROVED** [review]

**Pull request merged**